components/Login/LoginModal.js

```diff
@@ -134,8 +134,9 @@ class LoginModal extends React.Component {
     componentDidUpdate() {
         if (this.props.errorMessage.length > 2)
             $('.ui.form.signin').form('add errors', [this.props.errorMessage]);
-        console.log('componentDidUpdate:', this.props.errorMessage, ',', this.props.socialLoginError, ',', this.props.userid, ',', this.props.username, ',', this.state.isLoading);
+        // console.log('componentDidUpdate:', this.props.errorMessage, ',', this.props.socialLoginError, ',', this.props.userid, ',', this.props.username, ',', this.state.isLoading);
         if (localStorage.getItem(MODI) === 'login' && this.props.socialLoginError){
+            localStorage.setItem(MODI, 'login_failed');
```

Comment:
Is localstorage really needed for login?

Reply:
for social login yes. for normal login not. Could be improved later.